Userland/Libraries/LibJS/Runtime/Map.cpp

```diff
@@ -22,6 +22,60 @@ Map::~Map()
 {
 }
 
+void Map::map_clear()
+{
+    m_keys.clear();
+    m_entries.clear();
+}
+
+bool Map::map_remove(Value const& key)
```

Comment:
Since these are effectively the spec implementation of `Map.prototype.xxx` perhaps spec links would be good?
Not quite sure what would be best here. Perhaps only part of the spec text?

Because there is one detail which is now incorrect compared to the spec.

Reply:
I'm not sure if that makes sense, those prototype functions are always linked to in FooPrototype.cpp (and they are actually linked to in MapPrototype.cpp), these are more like AK/HashMap.h

I'll link to it if you think it makes sense though :shrug:

Reply:
Yeah this is a tricky one, I'm not sure what is the best here. 
I think not having them is fine since we don't strictly implement the spec anyway.
But I don't know how clear that is for people looking at Map code for the first time.